### DISCUSSION TITLE
Slides: Getting More Value from your OSPO

### DISCUSSION THREAD

**Discussion by geekygirldawn** [todogroup/ospology#375] [OSPO Stories]
PDF Slide link for my presentation at OSPOlogy live in Frankfurt October 10 and 11, 2023: [Getting More Value from your OSPO](https://github.com/todogroup/ospology/files/12878009/sm_Getting_More_Value_from_your_OSPO_OSPOlogy_DE.pdf)

I had a lovely time! Thank you to the SAP folks for being amazing hosts, and thank you to everyone for all of the interesting discussions. I'm looking forward to the next one!

**Comment by souravcipher** [comment]
Thanks for sharing the slides @geekygirldawn!!
